packages/meteor-otel/server/config.js

```
@@ -0,0 +1,48 @@
+ * - OTEL_EXPORTER_OTLP_ENDPOINT: Base OTLP endpoint (default: 'http://localhost:4318')
+ * - OTEL_EXPORTER_OTLP_TRACES_ENDPOINT: Specific traces endpoint
+ * - OTEL_EXPORTER_OTLP_METRICS_ENDPOINT: Specific metrics endpoint
+ * - OTEL_METRICS_EXPORT_INTERVAL_MS: Metrics export interval (default: 1000)
+ * - OTEL_HOST_METRICS_ENABLED: Set to '0' to disable host metrics (default: enabled)
+ * - OTEL_RUNTIME_METRICS_ENABLED: Set to '0' to disable runtime metrics (default: enabled)
+ */
+
+export function getConfig() {
+  const serviceName = process.env.OTEL_SERVICE_NAME || 'meteor-app';
+  const debug = process.env.OTEL_DEBUG === '1';
+  const exportIntervalMs = Number(process.env.OTEL_METRICS_EXPORT_INTERVAL_MS || 1000);
```

ESLint

[error] Replace `process.env.OTEL_METRICS_EXPORT_INTERVAL_MS·||·1000` with `⏎····process.env.OTEL_METRICS_EXPORT_INTERVAL_MS·||·1000,⏎··`

(prettier/prettier)